backend/src/serverless/integrations/workers/stripeWebhookWorker.ts

```diff
@@ -76,13 +77,17 @@ export const processStripeWebhook = async (message: any) => {
         process.exit(1)
       } else {
         log.info({ tenantId }, `Tenant found - updating tenant plan to Growth plan!`)
-        await tenant.update({
-          plan: Plans.values.growth,
-          isTrialPlan: false,
-          trialEndsAt: null,
-          stripeSubscriptionId: stripeWebhookMessage.data.object.subscription,
-          planSubscriptionEndsAt: moment(subscriptionEndsAt, 'X').toISOString(),
-        })
+        await TenantRepository.update(
+          tenantId,
+          {
+            plan: Plans.values.growth,
+            isTrialPlan: false,
+            trialEndsAt: null,
+            stripeSubscriptionId: stripeWebhookMessage.data.object.subscription,
```

Comment:
Add to lodash.pick inside tenantrepo.update so these two properties get actually updated